### ISSUE TITLE
'GADAdMobExtras.h' file not found on iOS

### ISSUE THREAD

**Issue by gclsoft** [googleads/googleads-mobile-plugins#38]
GADAdMobExtras.h can't be found when building on iOS, this file also can't be found in https://github.com/googleads/googleads-mobile-ios-examples

**Comment by PAHeartBeat** [comment]
same Issue, can't be found bellow files when building on iOS, XCode throws error,
GADAdMobExtras.h 
GADAdSize.h

I tried to follow this document "https://developers.google.com/mobile-ads-sdk/docs/admob/ios/rel-notes" but after that its XCode throws error of missing "libiPhone-lib" in Libraries folder

**Comment by ericleich** [comment]
Two questions:

1) Which version of Unity are you running?
2) Are you adding the `-ObjC` flag to the `Other linker flags` in Xcode?

**Comment by lsmoura** [comment]
1) I'm using Unity 4.6.1f1 and I'm having the same issue.
2) I have not touched the XCode project other than setting the "Team" for signature.

**Comment by PAHeartBeat** [comment]
1. I am using Unity 4.6.1p4
2. I have allrady Added th -ObjC falg in Other liker Flags.


Update:
I have solved this issue with little diffrent way
This demp project not contains Base AdMob iOS SDK so I added it as manual SDK setup as per link "https://developers.google.com/mobile-ads-sdk/docs/admob/ios/quick-start#manually_using_the_sdk_download", After follow all step as per link it's gives me a another xcode error posted here in Google Support fourm "https://groups.google.com/forum/#!category-topic/google-admob-ads-sdk/game-engines/yyiDQo3OPq4"

Later I also solved reported error by Unity fourm help "http://answers.unity3d.com/questions/538363/error-when-i-updated-to-xcode-5-to-get-import-to-i.html"

Report error was xocde report a file "liPhone-lib" file is missing in Libraries folder, I have solved it as per mattssono comment on his accepted answer as bellow step.

Hi Jason, sure.
1. Right-click libiPhone-lib.a in Libraries and click Delete
2. Click Remove Reference in the pop-up
3. Right-click the Libraries folder in Xcode and click Add Files to ... (CMD + Option + A)
4. Select the libiPhone-lib.a file and add it
5. This should create a working path to Libraries in Library Search Paths and make the Linker error go away.
